Example/Backpack NativeUITests/DialogUITest.swift

```diff
@@ -70,7 +70,9 @@ class DialogUITest: XCTestCase {
         saveScreenshot(named: "Dialog Controller Normal")
 
         XCTContext.runActivity(named: "Act") { _ in
-            dialogScrimView.tap()
+            let dialogScrimViewCoordinate: XCUICoordinate =
```

Comment:
Had to update this test, as iOS 13.2 navigation bars work slightly differently, and the click was being intercepted by the nav-bar